cshsms/settings_travis.py

```diff
@@ -14,3 +16,6 @@
 REMOTE = {'user': 'travis',
           'keyfile': '~/.ssh/travis.pem',
           'host': 'localhost'}
+
+os.getenv("TEXTLOCAL_API")
+os.getenv("TEXTLOCAL_PRIMARY_ID")
```

Comment:
These need to be assigned to variables (e.g., `TEXTLOCAL_API = os.getenv("TEXTLOCAL_API")`).